adm/style/acp_phpbb_ideas.html

```diff
@@ -28,16 +28,17 @@
 			<dd>{options.CONTENT}</dd>
 			<!-- IF options.KEY == 'ideas_poster_id' --><dd>[ <a href="{U_FIND_USERNAME}" onclick="find_username(this.href); return false;">{L_FIND_USERNAME}</a> ]</dd><!-- ENDIF -->
 		</dl>
+		<!-- IF options.KEY == 'ideas_base_url' -->
```

Comment:
I think using that cfg_array would have been fine if we used all of it's built in methods and validations. But we're not, we're customizing each option in it, so we're better off just using a more traditional acp module, like in the other extensions :smile: